### PR TITLE
[MDS-174] Dockerize and host on Azure Apps

### PR DESCRIPTION
**Pull request by edsp-david**
- Moved a couple dependencies to devDependencies to try to reduce container size

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
```
Comment (lipsotiko) [inline build.sh]: chmod +x ./build.sh

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
```
Comment (lipsotiko) [inline deploy.sh]: chmod +x ./deploy.sh

File: build.sh (hunk 1 above)
Comment (ddhogan) [reply build.sh]: Added to latest commit

File: deploy.sh (hunk 2 above)
Comment (ddhogan) [reply deploy.sh]: Added to latest commit

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+chmod +x ./build.sh
```
Comment (lipsotiko) [inline build.sh]: not what I mean't :)

**Pull request merged**